### PR TITLE
Fix snake kebab cases with uppercased First Letter

### PR DESCRIPTION
**Pull request by menthol**

### CODE REVIEW

File: src/util/kebab/index.js [hunk 1]
```diff
@@ -5,4 +5,4 @@
  * @returns {string} kekab-case formatted string
  * @private
  */
-export default s => s.replace(/[A-Z\u00C0-\u00D6\u00D8-\u00DE]/g, match => '-' + match.toLowerCase())
+export default s => s.charAt(0).toLowerCase() +  s.slice(1).replace(/[A-Z\u00C0-\u00D6\u00D8-\u00DE]/g, match => '-' + match.toLowerCase())
```
Comment (wopian) [inline src/util/kebab/index.js]: Remove extra space before `s.slice(1)`

File: src/util/snake/index.js [hunk 2]
```diff
@@ -5,4 +5,4 @@
  * @returns {string} snake_case formatted string
  * @private
  */
-export default s => s.replace(/[A-Z\u00C0-\u00D6\u00D8-\u00DE]/g, match => '_' + match.toLowerCase())
+export default s => s.charAt(0).toLowerCase() +  s.slice(1).replace(/[A-Z\u00C0-\u00D6\u00D8-\u00DE]/g, match => '_' + match.toLowerCase())
```
Comment (wopian) [inline src/util/snake/index.js]: Remove extra space before `s.slice(1)`

**Comment by menthol** [comment]
Fixed

**Comment by wopian** [comment]
Released in `4.2.1`

**Pull request merged**